{
  "schema_version": "1.4.0",
  "id": "GHSA-5h2q-4hrp-v9rr",
  "modified": "2022-05-17T05:12:01Z",
  "published": "2022-05-17T05:12:01Z",
  "aliases": [
    "CVE-2012-3444"
  ],
  "details": "The get_image_dimensions function in the image-handling functionality in Django before 1.3.2 and 1.4.x before 1.4.1 uses a constant chunk size in all attempts to determine dimensions, which allows remote attackers to cause a denial of service (process or thread consumption) via a large TIFF image.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2012-3444"
    },
    {
      "type": "WEB",
      "url": "https://www.djangoproject.com/weblog/2012/jul/30/security-releases-issued/"
    },
    {
      "type": "WEB",
      "url": "http://www.debian.org/security/2012/dsa-2529"
    },
    {
      "type": "WEB",
      "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2012:143"
    },
    {
      "type": "WEB",
      "url": "http://www.openwall.com/lists/oss-security/2012/07/31/1"
    },
    {
      "type": "WEB",
      "url": "http://www.openwall.com/lists/oss-security/2012/07/31/2"
    },
    {
      "type": "WEB",
      "url": "http://www.ubuntu.com/usn/USN-1560-1"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-119"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2012-07-31T17:55:00Z"
  }
}